### PR TITLE
Adaptation to blueprint + Update to Python 3.10

### PR DESCRIPTION
**Pull request by Karko93**
- Embedded Python blueprint into pyflexplot
- Updated Readme
- Updated to Python 3.10.12
- Updated reference material for the slow tests

### CODE REVIEW

**Comment by clairemerker** [comment]
The pinned installation is not working for me. I tried this on balfrin `tools/setup_env.sh -m`
and got this error:

```
Could not solve for environment specs
Encountered problems while solving:
  - package libsqlite-3.43.0-h2797004_0 requires libzlib >=1.2.13,<1.3.0a0, but none of the providers can be installed

The environment can't be solved, aborting the operation
```

File: environment.yml
Comment (clairemerker) [inline environment.yml:1]: This file can be removed I think.

File: tests/slow/test_pyflexplot/test_plots/ref_cosmo-1_affected_area.py
Comment (clairemerker) [inline tests/slow/test_pyflexplot/test_plots/ref_cosmo-1_affected_area.py:985]: Might be worth removing code duplication by defining identical blocks only once. This would also reduce the amount of maintenance work and the size of the diffs in further PRs... (this comment is valid for all files changed in `tests/slow`)

File: tools/find_py_toml.sh
Comment (clairemerker) [inline tools/find_py_toml.sh:1]: What is this script needed for? I can't find anything about it in the README.

File: README.md
Comment (clairemerker) [inline README.md:26]: This "getting started" focuses on development. Maybe it would be helpfull to have a quickstart section for people that just want to use it? So pinned dependencies, non-editable installation,...

File: README.md
Comment (clairemerker) [inline README.md:62]: Is Stefan still package admin?

File: README.md
Comment (clairemerker) [inline README.md:133]: This part needs revising still pyflexplot does not include `mutable_number.py` anymore

File: README.meteoswiss.md
Comment (clairemerker) [inline README.meteoswiss.md:1]: Why is there a separate README.meteoswiss.md? Could it be refered to at some point in the REAMDE.md?

**Review by clairemerker — CHANGES_REQUESTED** [review]
I reviewed a few things related to the blueprint update, but it is probably not an extensive review. I hope it helps nonetheless! 
Here are a few addtional general comments:
- As mentioned, the pinned installation fails. Is this known behaviour and will be fixed later?
- The unpinned installation worked, and the tests are all passing (with editable and non-editable installation)!
- The pre-commit hooks fail. You might want to fix that before merging the PR.

File: README.meteoswiss.md
Comment (Karko93) [reply README.meteoswiss.md:1]: As far as I know, it's an older ReadMe which describes some obsolete functionalities and some which are still relevant.
There is another PR in the pipeline in which this will be taken care of.

File: tools/find_py_toml.sh
Comment (Karko93) [reply tools/find_py_toml.sh:1]: I just discovered this as well.
This tool requires config files in .toml format to define your output. There are also a lot of tests implemented and therefore, config files defines. This script prints a list of all .toml files within the src folder in order to find them faster I guess?

File: environment.yml
Comment (Karko93) [reply environment.yml:1]: Done

File: tests/slow/test_pyflexplot/test_plots/ref_cosmo-1_affected_area.py
Comment (Karko93) [reply tests/slow/test_pyflexplot/test_plots/ref_cosmo-1_affected_area.py:985]: All files in tests/slow are dictionaries of references for some unit tests to compare the results with the generated output. These are automatically updated with help of a command.

File: tools/find_py_toml.sh
Comment (Karko93) [reply tools/find_py_toml.sh:1]: Shall I just keep it?

File: README.md
Comment (Karko93) [reply README.md:133]: Yeah, removed that part.

File: README.md
Comment (Karko93) [reply README.md:62]: I don't think so.

**Comment by Karko93** [comment]
Regarding pre-commit hooks and pinned environment:
There is another PR in the pipeline in which a new feature was implemented.
On this particular branch I started to work on the pre-commit errors and the pinned environment is working and tested there.
I'm awaiting customers approval right now and then I'm going to make the pull request which will resolve these issues.

File: tools/find_py_toml.sh
Comment (clairemerker) [reply tools/find_py_toml.sh:1]: If you as a developer find it useful, then keep it of course.

Comment (clairemerker) [reply tools/find_py_toml.sh:1]: But maybe document it somewhere

File: README.md
Comment (clairemerker) [reply README.md]: Yes perfect!

**Pull request merged**
